site/components/header/small-screen-nav.js

```diff
@@ -15,15 +15,20 @@ const trackAnalytics = title => () =>
   });
 
 export default class SmallScreenNav extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
     this.state = {
       navOpen: false,
     };
   }
 
+  state: {
+    navOpen: boolean
+  };
+  smallScreenNav: HTMLInputElement;
+
```

Comment:
Is it possible to move these type declarations above the constructor to the top of the class? Usually you have property declarations first by convention in object oriented languages.